src/formio.wizard.js

```diff
@@ -206,12 +206,27 @@ export class FormioWizard extends FormioForm {
   }
 
   hasButton(name, nextPage) {
+    // Check for and initlize button settings object
+    this.options.buttonSettings = this.options.buttonSettings || {};
```

Comment:
You could use [_.defaults](https://lodash.com/docs/4.17.4#defaults), something like this.

```
const _defaults = require('lodash/defaults');
...
...

this.options.buttonSettings = _defaults(this.options.buttonSettings, {
  showPrevious: true,
  showNext: true,
  showCancel: true
});
```